lib/profile/config.rb

```diff
@@ -33,6 +33,11 @@ def hunter_command
           ENV['flight_PROFILE_hunter_command'] ||
             config.hunter_command ||
             File.join(ENV.fetch('flight_ROOT', '/opt/flight/'), 'bin/flight hunter')
+        if !File.file?(File.join(command.split.join("/")))
```

Comment:
Why are you splitting the pathname? We're expecting for the user to give an absolute pathname and will be using the absolute pathname as they've written it. We should check the exact string they've given us, without modification.

Reply:
I thought the given pathname was written in the form "/dir1/dir2/dir3 hunter" and the split was used to obtain the directory and filename individually. I used split/join to convert it into the form "/dir1/dir2/dir3/hunter" so that `File.file?` can attempt to recognise it.

Reply:
The split is just a nicer way to present the command to `HunterCLI` as an array. You can have a look at that class to see what happens next.

You raise a good point in that the command can have extra bits. When running a dev version of hunter we would use `/path/to/executable`; when running in production it would be using `/opt/flight/bin/flight hunter`. In the first case, we can just use the string we're given; in the second case, we would want to ignore the `hunter` part, as it's an argument given to the `flight` executable. I would recommend changing this to:
```
if !File.file?(command.split(' ')[0])
```

This way, any extra arguments provided after the initial path to the executable are ignored when checking the filepath.

Reply:
I see, latest version should address this properly now.